### PR TITLE
RM-7580 Edge user directories are not cleaned up in EdgeBrowserSession.Dispose

### PR DESCRIPTION
**Pull request by a-ctor**

### CODE REVIEW

File: Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/EdgeConfiguration.cs [hunk 1]
```diff
@@ -61,26 +61,32 @@ public class EdgeConfiguration : BrowserConfigurationBase, IEdgeConfiguration
...
     public string DriverBinaryPath { get; }
     public string UserDirectoryRoot { get; }
+    public bool EnableUserDirectoryRootCleanup { get; }
     public string DownloadDirectory { get; }
 
     public EdgeConfiguration (
-        [NotNull] WebTestConfigurationSection webTestConfigurationSection)
-        : this (webTestConfigurationSection, s_edgeExecutable.Value)
+        [NotNull] WebTestConfigurationSection webTestConfigurationSection,
+        [NotNull] AdvancedEdgeOptions advancedEdgeOptions)
```
Comment (iam-mholle) [inline Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/EdgeConfiguration.cs:69]: This is a breaking change that will need documentation

File: Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/IEdgeConfiguration.cs [hunk 2]
```diff
@@ -43,6 +43,11 @@ public interface IEdgeConfiguration : IBrowserConfiguration
     [NotNull]
     string UserDirectoryRoot { get; }
 
+    /// <summary>
+    /// Defines whether the infrastructure should delete the given <see cref="UserDirectoryRoot"/>.
+    /// </summary>
+    bool EnableUserDirectoryRootCleanup { get; }
```
Comment (iam-mholle) [inline Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/IEdgeConfiguration.cs:49]: Also a potential breaking change.

**Review by iam-mholle — APPROVED** [review]

**Review by MichaelKetting — APPROVED** [review]

**Comment by a-ctor** [comment]
Closing this PR since the target branch has changed. New PR is #122

**Comment by iam-mholle** [comment]
FYI You can change the target branch of a PR.

**Comment by MichaelKetting** [comment]
> 
> 
> FYI You can change the target branch of a PR.

@iam-mholle the full text should have read: changed target branch and renamed source branch.

**Pull request closed without merging**